Userland/Libraries/LibWeb/Layout/GridFormattingContext.h

```diff
@@ -16,22 +16,30 @@ enum class GridDimension {
     Column
 };
 
+struct GridPosition {
+    size_t row;
+    size_t column;
+    inline bool operator==(GridPosition const&) const = default;
+};
+
 class OccupationGrid {
 public:
-    OccupationGrid(size_t column_count, size_t row_count);
-    OccupationGrid();
+    OccupationGrid(size_t columns_count, size_t rows_count)
+        : m_columns_count(columns_count)
+        , m_rows_count(rows_count) {};
+    OccupationGrid() {};
```

Comment:
If this constructor is used, `m_columns_count` and `m_rows_count` will be uninitialized.

Let's initialize them below:
```cpp
size_t m_column_count { 0 };
size_t m_row_count { 0 };
```